mibidata/constants.py

```diff
@@ -0,0 +1,15 @@
+"""Constants used in mibidata module.
+
+Copyright (C) 2020 Ionpath, Inc.  All rights reserved."""
+
+COLORS = {
```

Comment:
I like that these are defined in RGB instead of the hue. It's easier for me to recogize.